src/jsep.js

```diff
@@ -637,171 +825,116 @@ let jsep = function(expr) {
-jsep.toString = function() {
-	return 'JavaScript Expression Parser (JSEP) v' + jsep.version;
-};
-
-/**
- * @method jsep.addUnaryOp
- * @param {string} op_name The name of the unary op to add
- * @return jsep
- */
-jsep.addUnaryOp = function(op_name) {
-	max_unop_len = Math.max(op_name.length, max_unop_len);
-	unary_ops[op_name] = 1;
-	return this;
-};
-
-/**
- * @method jsep.addBinaryOp
- * @param {string} op_name The name of the binary op to add
- * @param {number} precedence The precedence of the binary op (can be a float)
- * @return jsep
- */
-jsep.addBinaryOp = function(op_name, precedence) {
-	max_binop_len = Math.max(op_name.length, max_binop_len);
-	binary_ops[op_name] = precedence;
-	return this;
-};
-
-/**
- * @method jsep.addIdentifierChar
- * @param {string} char The additional character to treat as a valid part of an identifier
- * @return jsep
- */
-jsep.addIdentifierChar = function(char) {
-	additional_identifier_chars.add(char);
-	return this;
-};
-
-/**
- * @method jsep.addLiteral
- * @param {string} literal_name The name of the literal to add
- * @param {*} literal_value The value of the literal
- * @return jsep
- */
-jsep.addLiteral = function(literal_name, literal_value) {
-	literals[literal_name] = literal_value;
-	return this;
-};
-
-/**
- * @method jsep.removeUnaryOp
- * @param {string} op_name The name of the unary op to remove
- * @return jsep
- */
-jsep.removeUnaryOp = function(op_name) {
-	delete unary_ops[op_name];
-	if (op_name.length === max_unop_len) {
-		max_unop_len = getMaxKeyLen(unary_ops);
-	}
-	return this;
-};
-
-/**
- * @method jsep.removeAllUnaryOps
- * @return jsep
- */
-jsep.removeAllUnaryOps = function() {
-	unary_ops = {};
-	max_unop_len = 0;
-
-	return this;
-};
-
-/**
- * @method jsep.removeIdentifierChar
- * @param {string} char The additional character to stop treating as a valid part of an identifier
- * @return jsep
- */
-jsep.removeIdentifierChar = function(char) {
-	additional_identifier_chars.delete(char);
-	return this;
-};
-
-
-/**
- * @method jsep.removeBinaryOp
- * @param {string} op_name The name of the binary op to remove
- * @return jsep
- */
-jsep.removeBinaryOp = function(op_name) {
-	delete binary_ops[op_name];
-
-	if (op_name.length === max_binop_len) {
-		max_binop_len = getMaxKeyLen(binary_ops);
-	}
-
-	return this;
-};
-
-/**
- * @method jsep.removeAllBinaryOps
- * @return jsep
- */
-jsep.removeAllBinaryOps = function() {
-	binary_ops = {};
-	max_binop_len = 0;
-
-	return this;
-};
-
-/**
- * @method jsep.removeLiteral
- * @param {string} literal_name The name of the literal to remove
- * @return jsep
- */
-jsep.removeLiteral = function(literal_name) {
-	delete literals[literal_name];
-	return this;
-};
-
-/**
- * @method jsep.removeAllLiterals
- * @return jsep
- */
-jsep.removeAllLiterals = function() {
-	literals = {};
-
-	return this;
-};
+}
+
+// Backward Compatibility (before adding the static fields):
+const jsep = expr => (new Jsep(expr)).parse();
```

Comment:
I figure this can be `Jsep.parse` or `expr => (new Jsep(expr)).parse()`?

Reply:
If it's `Jsep.parse` then the functions we're adding to it afterwards would be added to `Jsep.parse` as well.

Reply:
It could however be `expr => Jsep.parse(expr)`